week 5/README.md

```diff
@@ -190,3 +190,25 @@ Output
 1 2 3
 ```
 
+
+### Task 3 - WTF Cipher
+
+You are given a sentence and a keyword (e.g. `You are great` and keyword `cipher`). Your task is to encrypt this sentence following these rules:
+* the first symbol in the sentence e.g. `Y` will be shifted with the corresponding symbol of the keyword e.g. 'c'.
+* `Y` is 89 in ASCII code and `c` is 99 is ASCII code so the result is 188 in ASCII. [ASCII Table](http://www.ascii-code.com/)
+* every next letter of the sentence will be encoded with the next symbol of the keyword.
+* every time you get to the of the keyword start from the beginning of it again.
+* The keyword is only six letters long, so for the seventh letter of the sentence a `c` symbol is used again to encode.
+
+**Input:**
+
+```
+You are great
+cipher
+```
+
+**Output:**
+
+```
+¼ØåˆÆäÈ‰×ÚÊÓ×
```

Comment:
I just copied the symbols from the ASCII table.  I didn't tested it on my machine in C++ environment. Only in Javascript, so I suppose it depends on the current language.
What do you think is better to print out.